### PR TITLE
Missing translations

### PR DESCRIPTION
**Pull request by Ladias-Elie**

### CODE REVIEW

File: src/AppBundle/Resources/translations/messages.fr.yml [hunk 1]
```diff
@@ -113,3 +113,16 @@ search.help: >
...
+Open: Ouvert
+Closed: Fermé
+Contact us: Contactez nous
+Terms of service: Conditions d'utilisation
+Continue to payment: Paiement
+You don't have any restaurants yet: Vous n'avez pas encore de restaurants
+Create restaurant: Créer un restaurant
+Upload an image: Télécharger une image
+Default: Défaut 
+Serves cuisine: Cuisines proposées
```
Comment (Atala) [inline src/AppBundle/Resources/translations/messages.fr.yml]: I think `Type de plats` is better. (`Cuisine` stands for meal types as Burger, Italian, Greek, aso)

File: src/AppBundle/Resources/translations/messages.fr.yml [hunk 2]
```diff
@@ -113,3 +113,16 @@ search.help: >
...
 
 About: À propos
+Open: Ouvert
+Closed: Fermé
+Contact us: Contactez nous
+Terms of service: Conditions d'utilisation
+Continue to payment: Paiement
+You don't have any restaurants yet: Vous n'avez pas encore de restaurants
+Create restaurant: Créer un restaurant
+Upload an image: Télécharger une image
```
Comment (Atala) [inline src/AppBundle/Resources/translations/messages.fr.yml]: `Télécharger` is translation for `Download`. Exact translation would be `Télédéposer` but I think we can use `Uploader` in french too.

File: src/AppBundle/Resources/translations/messages.fr.yml [hunk 3]
```diff
@@ -113,3 +113,16 @@ search.help: >
...
+Default: Défaut 
+Serves cuisine: Cuisines proposées
+no order in progress: pas de commande en cours
```
Comment (Atala) [inline src/AppBundle/Resources/translations/messages.fr.yml]: Please capitalize for both languages.

**Comment by alexsegura** [comment]
Fixes #96

**Comment by alexsegura** [comment]
Thank you @Ladias-Elie 👍

**Pull request merged**